.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "dash-pipeline/SAI/SAI"]
 	path = dash-pipeline/SAI/SAI
-	url = https://github.com/reshmaintel/SAI.git
```

Comment:
something is not right this new branch revers some useful changes

Reply:
I reverted the changes